src/osd/ReplicatedBackend.cc

```diff
@@ -789,7 +789,8 @@ void ReplicatedBackend::be_deep_scrub(
   ScrubMap::object &o,
   ThreadPool::TPHandle &handle)
 {
-  dout(10) << __func__ << " " << poid << " seed " << seed << dendl;
+  dout(10) << __func__ << " " << poid << " seed " 
```

Comment:
no need to restore the iostream state by sending `std::dec` to it. `dout` creates a new `ceph::log::Entry` instance. for every `dout()`. the `ostream` under the hood is the `ceph::log::Entry::m_streambuf` variable, and they are not shared.